src/components/AuthForm/LoginForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import FormInput from "./FormInput";
+import FormButton from "@/components/AuthForm/FormButton";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import { useRouter } from "next/router";
+import { useAuthStore } from "@/stores/authStore";
+import { loginAPI } from "@/apis/authAPI";
+
+const LoginForm = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    mode: "onBlur",
+  });
+
+  const router = useRouter();
+  const [isLoading, setIsLoading] = useState(false);
+  const [mounted, setMounted] = useState(false);
+  const { login, isInitialized } = useAuthStore();
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  const onSubmit = async (data: { email: string; password: string }) => {
+    if (!mounted || !isInitialized) return;
+
+    try {
+      setIsLoading(true);
+
+      const response = await loginAPI(data);
+
+      login(response.accessToken);
+      const currentAuthState = useAuthStore.getState();
+      console.log("현재 상태:", {
+        isLoggedIn: currentAuthState.isLoggedIn,
+        token: currentAuthState.token,
+      });
+      alert("로그인 성공!");
+      router.push("/");
+    } catch (error) {
+      //   console.error("로그인 실패:", error);
+      alert("로그인 실패했습니다.");
```

Comment:
에러객체 안쓸때는 굳이 작성해 줄 필요가 없을 것 같아요.
```suggestion
    } catch {
      //   console.error("로그인 실패:", error);
      alert("로그인 실패했습니다.");
```